src/openfermion/ops/operators/symbolic_operator.py

```diff
@@ -618,15 +618,17 @@ def __next__(self):
         term, coefficient = next(self._iter)
         return self.__class__(term=term, coefficient=coefficient)
 
-    def isclose(self, other, tol=EQ_TOLERANCE):
+    def isclose(self, other, rel_tol=EQ_TOLERANCE, abs_tol=EQ_TOLERANCE):
```

Comment:
this is technically a breaking API change. are we ok with that? weigh confusion vs. code compatibility to keeping one of them named `tol`

Reply:
Yeah, maybe we could keep 'tol' as a deprecated parameter and set atol=tol if it exists.